tuna/tables_interface.py

```diff
@@ -26,28 +26,33 @@
 ###############################################################################
 """Module that encapsulates the DB representation for a library"""
 
+from typing import Any, Set, Union, Optional, Dict
 from tuna.dbBase.sql_alchemy import DbSession
+from tuna.miopen.db.miopen_tables import ConvolutionJob, BNJob, FusionJob
+from tuna.example.example_tables import Job
+from tuna.miopen.db.session import Session
+from tuna.example.session import SessionExample
 
 
 #pylint: disable=too-few-public-methods
 class DBTablesInterface():
   """Represents db tables interface class"""
 
-  def __init__(self, **kwargs):
+  def __init__(self, **kwargs: Dict[str, Dict[str, Any]]):
     """Constructor"""
     super().__init__()
-    allowed_keys = set(['session_id'])
+    allowed_keys: Set = set(['session_id'])
     self.__dict__.update((key, None) for key in allowed_keys)
 
-    #for pylint
-    self.job_table = None
-    self.session_id = None
-    self.session = None
+    self.job_table: Optional[Union[ConvolutionJob, BNJob, FusionJob,
```

Comment:
Simplify this to the parent class. These are all JobMixin.

Reply:
this is nice, didnt realize you can do this, simplifies things